### PR TITLE
Added RotationalStiffness unit

### PR DESCRIPTION
**Pull request by dschuermans**
Added a new unit for stiffness, particularly rotational stiffness:

[Rotational Stiffness (Wikipedia)](https://en.wikipedia.org/wiki/Stiffness#Rotational_stiffness)

Used in structural engineering.

Base unit is Newton meters per radian.
Supported prefixes are Mega and Kilo

### CODE REVIEW

File: UnitsNet.Tests/CustomCode/RotationalStiffnessTests.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+using System;
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class RotationalStiffnessTests : RotationalStiffnessTestsBase
+    {
+        protected override double NewtonMetersPerRadianInOneNewtonMeterPerRadian => 1D;
+        protected override double MeganewtonMetersPerRadianInOneNewtonMeterPerRadian => 1E-6D;
+        protected override double KilonewtonMetersPerRadianInOneNewtonMeterPerRadian => 1E-3D;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/RotationalStiffnessTests.cs]: Use this format: `1e-3`

File: UnitsNet/UnitDefinitions/RotationalStiffness.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "RotationalStiffness",
+  "BaseUnit": "NewtonMeterPerRadian",
+  "XmlDoc": "The resistance offered by an elastic body to deformation by means of rotation.",
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/RotationalStiffness.json]: Can add this: 
```
"XmlDocRemarks": "https://en.wikipedia.org/wiki/Stiffness#Rotational_stiffness",
```

File: UnitsNet/UnitDefinitions/RotationalStiffness.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    {
+      "SingularName": "NewtonMeterPerRadian",
+      "PluralName": "NewtonMetersPerRadian",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Kilo", "Mega" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "Nm/rad" ]
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/RotationalStiffness.json]: We  have a [guideline](https://github.com/angularsen/UnitsNet/blob/master/CONTRIBUTING.md#unit-abbreviations) to use this notation `N·m`. It's also how [`Torque`](https://github.com/angularsen/UnitsNet/blob/master/UnitsNet/UnitDefinitions/Torque.json#L41) has defined it.

You can keep both variations if you want to be able to `Parse()` both, but the first one is used by `ToString()`:
```
"Abbreviations": [ "N·m/rad", "Nm/rad" ]
```

**Comment by angularsen** [comment]
Forgot, you can also add operator overloads for arithmetic with other quantities. 
The convention is that the left-hand quantity type implements the operator overload.

Example:
https://github.com/angularsen/UnitsNet/blob/master/UnitsNet/CustomCode/Quantities/Torque.extra.cs

You can add these three if they make sense to you and seem useful:
```csharp
var angle = Angle.FromDegrees(30);
var torque = Torque.FromNewtonMeters(10);
var rotstiff = RotationalStiffness.FromNewtonMeterPerRadian(1);

RotationalStiffness rs = torque / angle; // CustomCode/Torque.extra.cs
Torque t = rotstiff * angle; // CustomCode/RotationalStiffness.extra.cs
Angle a = torque / rotstiff; // CustomCode/Torque.extra.cs
```

**Comment by dschuermans** [comment]
@angularsen updated code according to your comments. Also added the operators you suggested :)

File: UnitsNet/CustomCode/Quantities/RotationalStiffness.extra.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+// THE SOFTWARE.
+
+
+namespace UnitsNet
+{
+    // Windows Runtime Component has constraints on public types: https://msdn.microsoft.com/en-us/library/br230301.aspx#Declaring types in Windows Runtime Components
+    // Public structures can't have any members other than public fields, and those fields must be value types or strings.
+    // Public classes must be sealed (NotInheritable in Visual Basic). If your programming model requires polymorphism, you can create a public interface and implement that interface on the classes that must be polymorphic.
+#if WINDOWS_UWP
+    public sealed partial class RotationalStiffness
```
Comment (angularsen) [inline UnitsNet/CustomCode/Quantities/RotationalStiffness.extra.cs]: Copy & paste error, I fixed it for you. Class name was still `Torque` for UWP.

**Comment by angularsen** [comment]
Nuget 3.93 on the way out, with this change.

**Pull request merged**